modules/mobile/spec/requests/mobile/v0/user/addresses_spec.rb

```diff
@@ -7,8 +7,9 @@
 
   let!(:user) { sis_user }
 
-  describe 'update endpoints' do
+  describe 'update endpoints', :skip_va_profile_user do
     Flipper.disable(:va_v3_contact_information_service)
+    Flipper.disable(:remove_pciu)
```

Comment:
Do we have tests that support both scenarios for the `remove_pciu` feature toggle?

Reply:
@LindseySaari Not for mobile.. yet. Mobile will be tested later